### ISSUE TITLE
Error raised by `galsim.hsm.EstimateShear` in the metrics part

### ISSUE THREAD

**Issue by ismael-mendoza** [LSSTDESC/BlendingToolKit#156]
```
Unphysical situation: galaxy convolved with PSF is smaller than PSF!
Error: adaptive moment failed
Unphysical situation: galaxy convolved with PSF is smaller than PSF!
```

Not sure what could be causing this since every galaxy is convolved by the PSF, any suggestions appreciated!

**Comment by thuiop** [comment]
Ok, I've found it ; it happens when SExtractor makes spurious detections/only detects a part of a galaxy which is barely above the noise. So it actually is not a real problem ; we can remove the errors if you want (or maybe add the numbers of the blends which are concerned ?). 
Also, I realized that the "new" matching you added (using linear_sum_assignment) will match detected and true galaxies no matter what ; there is no maximum distance between the two, which does not sound good.
